VideoWeb/VideoWeb.Common/Models/Conference.cs

```diff
@@ -209,5 +209,38 @@ public void RemoveTelephoneParticipant(Guid id)
         {
             TelephoneParticipants.RemoveAll(x => x.Id == id);
         }
+
+        public List<Guid> GetHosts()
```

Comment:
Because we use the term hosts in video web already, would you mind renaming the method to `GetNonScreenedParticipants()`
Hosts according the the video supplier means a different thing here but for us it would be anyone not screened

Reply:
I've updated it to `GetNonScreenedParticipantsAndEndpoints()` as it contains both - happy with that?